src/compiler/diagnosticMessages.json

```diff
@@ -3720,6 +3720,10 @@
         "category": "Error",
         "code": 8028
     },
+    "JSDoc '...' must appear on the last @param tag of a function that uses 'arguments'.": {

```

Comment:
This sounds like it's complaining about `...` in an invalid location. Would write this backwards: the last tag must have `...`.